### PR TITLE
Simple env

### PR DESCRIPTION
**Pull request by captainrandom**
# Motivation and context
Easing the on-boarding by dockerizing the environments and replicating the commands in the makefile to also have a docker version.

# What I did

I also noticed that load_dataset("data/planting/TreePlanting_Medians.shp") didn't load correctly, so I tried setting the crs value to make it load, since I'm not 100% sure what the value was before, I'm probably going to need some help validating the python code change.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,40 +1,72 @@
...
-	  | python pruning_planting.py \
-	  | node download-images.js \
-	  | node split-trees.js build/data
+	  | node $(node-dir)/parse-trees.js \
+	  | python $(python-dir)/pruning_planting.py \
+	  | node $(node-dir)/download-images.js \
+	  | node $(node-dir)/split-trees.js build/data
+
+release-docker: setup
+	curl 'https://data.smgov.net/resource/w8ue-6cnd.csv?$$limit=50000' \
```
Comment (theswerd) [inline Makefile]: This won't work, the proper query only has 1 $

File: Makefile [hunk 2]
```diff
@@ -1,40 +1,72 @@
...
+
+img-test-docker: setup
+	cat data/trees.csv \
+	  | docker run -i --rm -v $$(pwd)/data:/public-tree-map/data public-tree-map-node:latest node /public-tree-map/parse-trees.js \
+	  | docker run -i --rm -v $$(pwd)/data:/public-tree-map/data public-tree-map-node:latest node /public-tree-map/download-images.js \
+	  | docker run -i --rm -v $$(pwd)/data:/public-tree-map/data -v $$(pwd)/build:/public-tree-map/build public-tree-map-node:latest node /public-tree-map/split-trees.js build/data
 
 # Runs the pipeline, but skips downloading images
 no-images: setup
 	curl 'https://data.smgov.net/resource/w8ue-6cnd.csv?$$limit=50000' \
```
Comment (theswerd) [inline Makefile]: This won't work, proper link is 'https://data.smgov.net/resource/w8ue-6cnd.csv?$limit=5000'

File: Makefile [hunk 3]
```diff
@@ -1,40 +1,72 @@
...
-	  | node parse-trees.js \
-	  | node split-trees.js build/data
+	  | node $(node-dir)/parse-trees.js \
+	  | node $(node-dir)/split-trees.js build/data
+
+no-images-docker: setup
+	curl 'https://data.smgov.net/resource/w8ue-6cnd.csv?$$limit=50000' \
```
Comment (theswerd) [inline Makefile]: This won't work, proper link is 'https://data.smgov.net/resource/w8ue-6cnd.csv?$limit=5000'

File: Makefile (hunk 1 above)
Comment (captainrandom) [reply Makefile]: It Runs

Please see the command I just ran with the double "$$" in the make file, but the results in the output are as expected.

Why is this the case:

This stack overflow reference should explain why this happens:
https://stackoverflow.com/questions/19589957/makefile-and-use-of#comment29075486_19589957

File: Makefile (hunk 2 above)
Comment (captainrandom) [reply Makefile]: See comment above

File: Makefile (hunk 3 above)
Comment (captainrandom) [reply Makefile]: See comment above

**Comment by emillipede** [comment]
hi @captainrandom 

when I merged the PR, circle ci failed in make release. I forgot that there's a [test branch](https://github.com/Public-Tree-Map/public-tree-map-data-pipeline/compare/test-circleci) for circle ci, I should've merged the PR there first - next time

please take a look at the error and let me know what you think. you should be able to log in to circle ci with your github account, but I've copied the error message below:

```
mkdir -p build/data
curl 'https://data.smgov.net/resource/w8ue-6cnd.csv?$limit=50000' \
  | node ./node/src/parse-trees.js \
  | python ./python/src/pruning_planting.py \
  | node ./node/src/download-images.js \
  | node ./node/src/split-trees.js build/data
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0internal/modules/cjs/loader.js:550
    throw err;
    ^

Error: Cannot find module 'csv-parse/lib/sync'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at Object.<anonymous> (/root/project/node/src/split-trees.js:12:18)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
internal/modules/cjs/loader.js:550
    throw err;
    ^

Error: Cannot find module 'csv-parse/lib/sync'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at Object.<anonymous> (/root/project/node/src/parse-trees.js:51:18)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
internal/modules/cjs/loader.js:550
    throw err;
    ^

Error: Cannot find module 'got'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at Object.<anonymous> (/root/project/node/src/download-images.js:28:18)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (23) Failed writing body (0 != 15011)
Traceback (most recent call last):
  File "./python/src/pruning_planting.py", line 130, in <module>
    trees = pd.read_json(infile)
  File "/root/miniconda/envs/***************/lib/python3.7/site-packages/pandas/io/json/json.py", line 427, in read_json
    result = json_reader.read()
  File "/root/miniconda/envs/***************/lib/python3.7/site-packages/pandas/io/json/json.py", line 537, in read
    obj = self._get_object_parser(self.data)
  File "/root/miniconda/envs/***************/lib/python3.7/site-packages/pandas/io/json/json.py", line 556, in _get_object_parser
    obj = FrameParser(json, **kwargs).parse()
  File "/root/miniconda/envs/***************/lib/python3.7/site-packages/pandas/io/json/json.py", line 652, in parse
    self._parse_no_numpy()
  File "/root/miniconda/envs/***************/lib/python3.7/site-packages/pandas/io/json/json.py", line 871, in _parse_no_numpy
    loads(json, precise_float=self.precise_float), dtype=None)
ValueError: Expected object or value
make: *** [Makefile:10: release] Error 1
Exited with code 2
```

(circle ci also failed at apt-get update, but it'll work if the line "run: apt-get update" is added to config.yml - see the change in that same [test branch](https://github.com/Public-Tree-Map/public-tree-map-data-pipeline/compare/test-circleci))

**Pull request merged**